tests/unit/OptionsListUtilsTest.ts

```diff
@@ -424,38 +424,12 @@ describe('OptionsListUtils', () => {
 
     it('getSearchOptions()', () => {
         // When we filter in the Search view without providing a searchValue
-        let results = OptionsListUtils.getSearchOptions(OPTIONS, '', [CONST.BETAS.ALL]);
+        const results = OptionsListUtils.getSearchOptions(OPTIONS, '', [CONST.BETAS.ALL]);
         // Then the 2 personalDetails that don't have reports should be returned
         expect(results.personalDetails.length).toBe(2);
 
         // Then all of the reports should be shown including the archived rooms.
         expect(results.recentReports.length).toBe(Object.values(OPTIONS.reports).length);
-
-        // When we filter again but provide a searchValue
-        results = OptionsListUtils.getSearchOptions(OPTIONS, 'spider');
-
-        // Then only one option should be returned and it's the one matching the search value
-        expect(results.recentReports.length).toBe(1);
-        expect(results.recentReports[0].login).toBe('peterparker@expensify.com');
-
-        // When we filter again but provide a searchValue that should match multiple times
-        results = OptionsListUtils.getSearchOptions(OPTIONS, 'fantastic');
-
-        // Value with latest lastVisibleActionCreated should be at the top.
-        expect(results.recentReports.length).toBe(2);
-        expect(results.recentReports[0].text).toBe('Mister Fantastic');
-        expect(results.recentReports[1].text).toBe('Mister Fantastic, Invisible Woman');
-
-        return waitForBatchedUpdates()
-            .then(() => Onyx.set(ONYXKEYS.PERSONAL_DETAILS_LIST, PERSONAL_DETAILS_WITH_PERIODS))
-            .then(() => {
-                const OPTIONS_WITH_PERIODS = OptionsListUtils.createOptionList(PERSONAL_DETAILS_WITH_PERIODS, REPORTS);
-                // When we filter again but provide a searchValue that should match with periods
-                results = OptionsListUtils.getSearchOptions(OPTIONS_WITH_PERIODS, 'barry.allen@expensify.com');
-                // Then we expect to have the personal detail with period filtered
-                expect(results.recentReports.length).toBe(1);
-                expect(results.recentReports[0].text).toBe('The Flash');
-            });
     });
```

Comment:
The getSearchOptions function is still in use, could you please explain why it's ok to remove these tests?

Reply:
I see you added some tests at the bottom which look similar, but again I'm having trouble determining what has changed. Maybe with more context it will be clear to me.